spec/ics-18-relayer-algorithms/README.md

```diff
@@ -0,0 +1,81 @@
+---
+ics: 18
+title: Relayer Algorithms
+stage: draft
+category: ibc-misc
+author: Christopher Goes <cwgoes@tendermint.com>
+created: 2019-03-07
+modified: 2019-04-15
+---
+
+## Synopsis
+
+Relayer algorithms are the "physical" connection layer of IBC — off-chain processes responsible for relaying data between two chains running the IBC protocol by scanning the state of each chain, constructing appropriate datagrams, and executing them on the opposite chain as allowed by the protocol.
+
+## Specification
+
+### Motivation
+
+In the IBC protocol, a blockchain can only record the *intention* to send particular data to another chain. Physical datagram relay must be performed by off-chain infrastructure. This standard defines the concept of a *relayer* algorithm, executable by an off-chain process with the ability to query chain state, to perform this relay.
+
+### Definitions
+
+A *relayer* is an off-chain process with the ability to read the state of and submit transactions to some set of ledgers utilizing the IBC protocol.
+
+### Desired Properties
```

Comment:
These properties are not necessarily telling us what we expect from relayer. It's more telling us what other part don't expect from relayers.

Reply:
Hmm, I suppose - these are desired properties of how we construct the relayer algorithm (e.g. permissionless - all datagrams are verified by the blockchain state machine itself). Some are a bit vague but I think still worth listing.

What properties do you think are in scope here?